tests/integration/functions.py

```diff
@@ -471,3 +471,114 @@ def floating_point_add(a_sign, a_exponent, a_mantissa, b_sign, b_exponent, b_man
     yield c_sign
     yield c_exponent
     yield c_mantissa
+
+
+def seven_seg(n: int) -> int:
+    """
+    Decimal digit to 7 segment display
+    0 means on, 1 means off
+    """
+    if n == 0:
+        return 192
+    elif n == 1:
+        return 249
+    elif n == 2:
+        return 164
+    elif n == 3:
+        return 176
+    elif n == 4:
+        return 153
+    elif n == 5:
+        return 146
+    elif n == 6:
+        return 130
+    elif n == 7:
+        return 248
+    elif n == 8:
+        return 128
+    elif n == 9:
+        return 144
+    else:
+        return 0
```

Comment:
Turn this into an array? Or is this what you're comparing this the generated code

Reply:
I don't support arrays right now, so I think this is the best I can do.